src/librustc_trans/context.rs

```diff
@@ -322,6 +322,7 @@ unsafe fn create_context_and_module(sess: &Session, mod_name: &str) -> (ContextR
     let llvm_target = sess.target.target.llvm_target.as_bytes();
     let llvm_target = CString::new(llvm_target).unwrap();
     llvm::LLVMRustSetNormalizedTarget(llmod, llvm_target.as_ptr());
+    llvm::LLVMRustSetModulePIELevel(llmod);
```

Comment:
Ok, awesome! So long as we're doing the same as before (whatever that PIELevel is), sounds good to me